Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -3791,6 +3792,83 @@ def com_google_fonts_check_name_family_and_style_max_length(ttFont):
     yield PASS, "All name entries are good."
 
 
+@check(
+    id='com.google.fonts/check/has_unacceptable_control_chars',
+    conditions=['are_ttf']
+)
+def com_google_fonts_check_has_unacceptable_control_chars(ttFonts):
+  """Does font file include unacceptable control character glyphs?
+
+  Args:
+      ttFonts (list): list of one or more fontTools.ttLib.TTFont objects
+  """
+  # list of unacceptable control character glyph names
+  # definition includes the entire control character Unicode block except:
+  #    - .null (U+0000)
+  #    - CR (U+000D)
+  unacceptable_cc_list = [
+      "uni0001",
+      "uni0002",
+      "uni0003",
+      "uni0004",
+      "uni0005",
+      "uni0006",
+      "uni0007",
+      "uni0008",
+      "uni0009",
+      "uni000A",
+      "uni000B",
+      "uni000C",
+      "uni000E",
+      "uni000F",
+      "uni0010",
+      "uni0011",
+      "uni0012",
+      "uni0013",
+      "uni0014",
+      "uni0015",
+      "uni0016",
+      "uni0017",
+      "uni0018",
+      "uni0019",
+      "uni001A",
+      "uni001B",
+      "uni001C",
+      "uni001D",
+      "uni001E",
+      "uni001F"
+  ]
+
+  # a dict with key:value of font path that failed check : list of unacceptable glyph names
+  failed_font_dict = {}
+
+  for tt in ttFonts:
+    font_failed = False
+    unacceptable_glyphs_in_set = []  # a list of unacceptable glyph names identified
+    glyph_name_set = set(tt["glyf"].glyphs.keys())
+    fontname = tt.reader.file.name
+
+    for unacceptable_glyph_name in unacceptable_cc_list:
+      if unacceptable_glyph_name in glyph_name_set:
+        font_failed = True
+        unacceptable_glyphs_in_set.append(unacceptable_glyph_name)
+
+    if font_failed:
+        failed_font_dict[fontname] = unacceptable_glyphs_in_set
+
+  if len(failed_font_dict) > 0:
+    unacceptable_cc_report_string = "The following unacceptable control characters were identified:{}".format(
+        os.linesep
```

Comment:
Please simply use `\n` here.

If there's a real need to use `os.linesep`, please open an issue to discuss that and eventually deal with it throughout the entire codebase.

Reply:
done!